oggm/tests/test_models.py

```diff
@@ -64,406 +62,434 @@ def teardown_module(module):
 DOM_BORDER = 80
 
 
-class TestInitFlowline(unittest.TestCase):
-
-    def setUp(self):
-        gdir = init_hef(border=DOM_BORDER)
-        self.testdir = os.path.join(get_test_dir(), type(self).__name__)
-        utils.mkdir(self.testdir, reset=True)
-        self.gdir = tasks.copy_to_basedir(gdir, base_dir=self.testdir,
-                                          setup='all')
-
-    def tearDown(self):
-        self.rm_dir()
+@pytest.fixture(scope='module')
+def io_model_factory():
+    glen_a = 2.4e-24
+    return lambda fls, **kwargs: FluxBasedModel(fls, glen_a=glen_a, **kwargs)
```

Comment:
Is this fixture *really* necessary?

Reply:
It really isn't. This is an artifact of a straight conversion from xunit class setups to analogous fixtures. Ideally, fixtures would define a certain test resource, rather than an entire setup specific to certain tests. This should be readdressed.